src/Microsoft.DotNet.Interactive.Http/HttpKernel.cs

```diff
@@ -124,28 +125,22 @@ async Task IKernelCommandHandler<SendValue>.HandleAsync(SendValue command, Kerne
 
     private Task SetValueAsync(string valueName, object value, Type? declaredType = null)
     {
-        if(value is (HttpVariableDeclarationAndAssignmentNode))
-        {
-            var variable = (HttpVariableDeclarationAndAssignmentNode) value;
-            if(variable.ValueNode is not null)
-            {
-                var bindingResult = variable.ValueNode.TryGetValue(BindExpressionValues);
-                if (bindingResult.IsSuccessful && bindingResult.Value is not null)
-                {
-                    _variables[valueName] = bindingResult.Value;
-                }
-            }
-            
-        } 
-        else
-        {
-            _variables[valueName] = value;
-        }
+        _variables[valueName] = value;
         return Task.CompletedTask;
     }
 
     async Task IKernelCommandHandler<SubmitCode>.HandleAsync(SubmitCode command, KernelInvocationContext context)
     {
+        string? doc = "";
```

Comment:
minor: Consider using string.Empty instead of ""

Also, looks like can the `?` be omitted since we only access the value of `doc` if `TryGetValue` succeds?

If yes, consider changing this to `var doc = string.Empty`. Otherwise, if we want to preserve the nullable string, we can probably just set it to null `string? doc = null`.